### PR TITLE
docs(contribution): improve info & format

### PR DESCRIPTION
**Pull request by Z3rio**
## Description
This PR improves the documentation for Contributing. Both in the way that it improves the information, but it also improves the style / format of the page at some places of the page.

## Motivation and Context
This is a needed pull request because it will increase the amount of contributors and make it easier for new contributors to make their first contribution.

## How Has This Been Tested?
Since this is just documentation, it has been tested locally on my computer, both with a dev server and after building.

## Markup:
N/A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/docs/src/pages/en/getting-started/contributing.md [hunk 1]
```diff
@@ -57,14 +65,28 @@ Some of our dependencies use [node-gyp](https://github.com/nodejs/node-gyp#insta
 
 Once you have everything installed, clone the repository:
 
+<v-tabs v-model="tab" color="primary">
```
Comment (MajesticPotatoe) [inline packages/docs/src/pages/en/getting-started/contributing.md:68]: This doesn't need to be fancy the standard code block is more than acceptable as is. The MD files should remain as close to a regular markup language as possible for purposes of translations and maintenance. The other small content changes are fine.

**Review by MajesticPotatoe — CHANGES_REQUESTED** [review]

File: packages/docs/src/pages/en/getting-started/contributing.md (hunk 1 above)
Comment (Z3rio) [reply packages/docs/src/pages/en/getting-started/contributing.md:68]: Okay, I will switch back then 👍🏼

Comment (Z3rio) [reply packages/docs/src/pages/en/getting-started/contributing.md]: There, this should be resolved.

File: packages/docs/src/pages/en/getting-started/contributing.md [hunk 2]
```diff
@@ -127,6 +128,8 @@ The documentation is located in `packages/docs` but also uses some files from `p
 
 If you want to see changes from Vuetify in the documentation you need to run `yarn build:lib` in the vuetify package before starting the documentation server.
 
+Please note that Vuetify v3's documentation is located on the `next` branch and not on the `master` branch.
```
Comment (MajesticPotatoe) [inline packages/docs/src/pages/en/getting-started/contributing.md:131]: wrap in `<alert type="info">`

File: packages/docs/src/pages/en/getting-started/contributing.md [hunk 3]
```diff
@@ -115,7 +116,7 @@ The **Playground** file is a cleanroom used for Vuetify development and is the r
     data: () => ({
       //
     }),
-  }
+  };
```
Comment (MajesticPotatoe) [inline packages/docs/src/pages/en/getting-started/contributing.md:119]: not needed (we generally don't use trailing ; in our codebase)

**Review by MajesticPotatoe — CHANGES_REQUESTED** [review]

**Review by MajesticPotatoe — APPROVED** [review]

**Pull request merged**